administrator/components/com_content/models/shared.php

```diff
@@ -0,0 +1,201 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_content
+ *
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+defined('_JEXEC') or die;
+require_once __DIR__ . '/articles.php';
+/**
+ * About Page Model
+ *
+ * @since  1.6
+ */
+class ContentModelShared extends ContentModelArticles
```

Comment:
Extend from JModelList instead, you don't need the ContentModelArticles as you are not doing anything with articles.